test/runtests.jl

```diff
@@ -2,6 +2,7 @@ using Test
 using CUSOLVERRF
 using SuiteSparse
 
+CUSOLVERRF.CUDA.versioninfo()
```

Comment:
Maybe remove this?

Reply:
Not sure. I think it might be not bad to have an indication in the logs of which artifact was used because I had several issues where I unintentionally loaded the wrong artifact (silently). This LocalPreferences.toml setup is a bit finicky and might change in the future.

Reply:
that works for me!